IN-CORE/maestro-service
# Postgres models with database connection

This task creates postgres database connection and performs CRUD operation through the defined SQLAlchemy models.

To test:
1. Create local postgres database named "maestro"
2. Set the connection string with credentials in `database.py: 7`
3. Run and test the apis on http://127.0.0.1:8000/docs
4. Verify if data is being fetched and inserted to postgres tables as expected.
